infra/tpu-pytorch-releases/artifacts_builds.tf

```diff
@@ -69,7 +69,7 @@ locals {
   versioned_builds_dict = {
     for b in var.versioned_builds :
     format("r%s_%s_%s",
-      b.package_version,
+      replace(b.package_version, "+", "_"),
```

Comment:
what does this line do?

Reply:
This replaces the `+` in the wheel name with a `_`. `+` is allowed in Python package names, but not in the trigger name or Docker image tag.

https://developer.hashicorp.com/terraform/language/functions/replace